packages/@react-spectrum/button/test/ButtonGroup.test.js

```diff
@@ -97,7 +97,288 @@ describe('ButtonGroup', function () {
     expect(group).toHaveAttribute('aria-disabled', 'true');
   });
 
-  it('SideNav handles single selection', function () {
+  it.each`
+    Name               | ComponentGroup   | Component       | props
+    ${'ButtonGroup'}   | ${ButtonGroup}   | ${ActionButton} | ${{}}
+  `('$Name shifts button focus on left/right keyboard arrow click (ltr + horizontal)', function ({ComponentGroup, Component, props}) {
+    let tree = render(
+      <Provider theme={theme} locale="de-DE">
+        <ComponentGroup {...props} >
+          <Component data-testid="button-1">Click me 1</Component>
+          <Component data-testid="button-2">Click me 2</Component>
+          <Component data-testid="button-3">Click me 3</Component>
+        </ComponentGroup>
+      </Provider>
+    );
+
+    let button1 = tree.getByTestId('button-1');
+    let button2 = tree.getByTestId('button-2');
+    let button3 = tree.getByTestId('button-3');
+
+    triggerPress(button1);
```

Comment:
triggerPress or just call focus()?
should we also check the rest of the tabIndices are -1?

Reply:
Eh I like triggerPress because it mimics what a user would do (aside from doing a TAB operation). I'll go ahead and add the -1 tabindex check for the other buttons here